process.c

```diff
@@ -4679,56 +4679,132 @@ rb_spawn(int argc, const VALUE *argv)
  *
  *  This method has potential security vulnerabilities if called with untrusted input;
  *  see {Command Injection}[rdoc-ref:command_injection.rdoc].
  *
- *  [<code>commandline</code>]
- *    command line string which is passed to the standard shell
- *  [<code>cmdname, arg1, ...</code>]
- *    command name and one or more arguments (no shell)
- *  [<code>[cmdname, argv0], arg1, ...</code>]
- *    command name, <code>argv[0]</code> and zero or more arguments (no shell)
+ *  Returns:
  *
- *  system returns +true+ if the command gives zero exit status,
- *  +false+ for non zero exit status.
- *  Returns +nil+ if command execution fails.
- *  An error status is available in <code>$?</code>.
+ *  - +true+ if the command exits with status zero.
+ *  - +false+ if the exit status is a non-zero integer.
+ *  - +nil+ if the command fails.
  *
- *  If the <code>exception: true</code> argument is passed, the method
- *  raises an exception instead of returning +false+ or +nil+.
+ *  Raises an exception (instead of returning +false+ or +nil+)
+ *  if keyword argument +exception+ is set to +true+.
  *
- *  The arguments are processed in the same way as
- *  for Kernel#spawn.
+ *  Assigns the command's error status to global variable <tt>$?</tt>.
  *
- *  The hash arguments, env and options, are same as #exec and #spawn.
- *  See Kernel#spawn for details.
+ *  The new process is created using the
+ *  {system system call}[https://pubs.opengroup.org/onlinepubs/9699919799.2018edition/functions/system.html];
+ *  it may inherit some of its environment from the calling program
+ *  (possibly including open file descriptors).
  *
- *     system("echo *")
- *     system("echo", "*")
+ *  Argument +env+, if given, is a hash that affects +ENV+ for the new process;
+ *  see {Execution Environment}[rdoc-ref:Process@Execution+Environment].
  *
- *  <em>produces:</em>
+ *  Argument +options+ is a hash of options for the new process;
+ *  see {Execution Options}[rdoc-ref:Process@Execution+Options].
+ *
+ *  The first required argument is one of the following:
+ *
+ *  - +command_line+ if it is a string,
+ *    and if it begins with a shell reserved word or special built-in,
+ *    or if it contains one or more metacharacters.
+ *  - +exe_path+ otherwise.
+ *
+ *  <b>Argument +command_line+</b>
+ *
+ *  \String argument +command_line+ is a command line to be passed to a shell;
+ *  it must begin with a shell reserved word, begin with a special built-in,
+ *  or contain meta characters:
+ *
+ *    system('echo')                             # => true  # Built-in.
+ *    system('if true; then echo "Foo"; fi')     # => true  # Shell reserved word.
+ *    system('date > /tmp/date.tmp')             # => true  # Contains meta character.
+ *    system('date > /nop/date.tmp')             # => false
+ *    system('date > date.tmp', exception: true) # Raises RuntimeError.
+ *
+ *  Assigns the command’s error status to global variable <tt>$?</tt>:
```

Comment:
This is a non ASCII character:

```suggestion
 *  Assigns the command's error status to global variable <tt>$?</tt>:
```

Reply:
Fixed in new commit.

Reply:
> Assigns the command’s error status to global variable

I believe it's thread local.

Reply:
> > Assigns the command’s error status to global variable
> 
> I believe it's thread local.

Removed  'global variable' (5 places).

Reply:
Thanks @BurdetteLamar!